### PR TITLE
83 Display answered questions bug

### PR DESCRIPTION
**Pull request by Alkhimovmv**
I fix this problem, now everything is displayed correctly, @AlexKay28 please check

### CODE REVIEW

**Comment by AlexKay28** [comment]
@Alkhimovmv 
can u explain what you did shortly please?

**Review by AlexKay28 — APPROVED** [review]

File: frontend/src/components/Question/Question.tsx [hunk 1]
```diff
@@ -61,14 +61,19 @@ const Question = () => {
...
         user_answer: user_answer
     }
+    const questionConfig = {
+      first_language: window.localStorage.getItem('firstLanguage'),
+      second_language: window.localStorage.getItem('secondLanguage'),
+      level: window.localStorage.getItem('level'),
+    }
     const session_token = window.localStorage.getItem('session_token')
-    api.patch('/answer', data, { headers: { session_token: `${session_token}`}})
+    await api.patch('/answer', data, { headers: { session_token: `${session_token}`}})
```
Comment (alantishin) [inline frontend/src/components/Question/Question.tsx:76]: it looks very strange
you can
```
const response = await api.patch('/answer', data, { headers: { session_token: `${session_token}`}})
const answerData = {answer: response.data.answer}
setAnswerData(answerData)

const reponse2 = await api.post('/question', questionConfig, { headers: { session_token: `${session_token}` } })
setQuestion(reponse2.data.question)
...
```

When use async/await it exclude necessity of using then

Comment (Alkhimovmv) [reply frontend/src/components/Question/Question.tsx]: Yes, you are absolutely right, I fixed it

**Comment by Alkhimovmv** [comment]
I made the /question request run after the /answer request. Typed variables in "Question" and "Answer" components. Also changed the table in accordance with the layout, that is, removed the title of the columns in the rows with previous answers

**Pull request merged**